adapta/security/clients/README.md

```diff
@@ -0,0 +1,27 @@
+# Authentication Clients for various platforms
+
+Currently supported:
+- HashiCorp Vault
+- Azure
+- Local (for unit tests)
+
+## Azure Client
+There are few important notes on the usage of this client for retrieving storage account credentials via `connect_storage` method. This method is also used by `delta_lake` loaders, thus it is important to understand how to configure your environment and account. 
+
+1. Environment Variables: The method first checks for mapped environment variables in a format `PROTEUS__STORAGEACCOUNTNAME_AZURE_STORAGE_ACCOUNT_KEY`, and if found, returns the account name and key.
```

Comment:
Reading the code, it seems the environment variable name depends on the storage account name. I don't think this is clear from the current description.
```suggestion
1. Environment Variables: The method first checks for mapped environment variables in a format `f'PROTEUS__{STORAGEACCOUNTNAME}_AZURE_STORAGE_ACCOUNT_KEY'`, and if found, returns the account name and key.
```